x-pack/plugins/licensing/server/licensing_config.ts

```diff
@@ -6,10 +6,9 @@
 
 import { schema, TypeOf } from '@kbn/config-schema';
 
-const SECOND = 1000;
 export const config = {
   schema: schema.object({
-    pollingFrequency: schema.number({ defaultValue: 30 * SECOND }),
+    pollingFrequency: schema.duration({ defaultValue: '30s' }),
```

Comment:
note: we don't implement deprecation in NP, but should by release date. The best I can do now is change configPath for the plugin